integration_tests/macros/edge_cases_to_ignore.sql

```diff
@@ -1,7 +1,6 @@
 {# Filters out edge cases in test data set which we haven't yet solved for #}
 {% macro edge_cases_to_ignore() %}
   user_id not in (
-    'stray page ping', -- Known unsolved issue https://github.com/snowplow/data-models/issues/92
-    'NULL domain_userid' -- Case when `domain_userid` is null but `domain_sessionid` is not null. Shouldn't happen. Will solve if it arises.
+    'stray page ping' -- Known unsolved issue https://github.com/snowplow/data-models/issues/92
```

Comment:
Given the change we made to deal with this, can we actually remove this now?

Reply:
Yes, will have to add the data to the affected tables but not a big deal, no edge cases to ignore anymore!

Reply:
I think it might not actually need to be added - because in theory we were guessing with these before but now we know they get ignored (unless the whole session is being excluded?)